resources/bundles/org.eclipse.core.resources/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.core.resources; singleton:=true
-Bundle-Version: 3.22.300.qualifier
+Bundle-Version: 3.23.0.qualifier
```

Comment:
I tend to agree with API tools here, according to the changed manifest you do not change API but only required bundles (that are not reexported).